### PR TITLE
Fix for flake8 warnings

### PR DESCRIPTION
**Pull request by praveenramoorthy**
Fix for flake8 warnings in galaxy

### CODE REVIEW

File: plugins/modules/dcnm_network.py [hunk 1]
```diff
@@ -1356,11 +1349,9 @@ def update_ms_fabric(self, diff):
             for node in list_elem['lanAttachList']:
                 node['fabric'] = self.sn_fab[node['serialNumber']]
 
-
     def push_to_remote(self, is_rollback=False):
 
         path = '/rest/top-down/fabrics/{}/networks'.format(self.fabric)
-        bulk_create_path = '/rest/top-down/bulk-create/networks'
```
Comment (mikewiebe) [inline plugins/modules/dcnm_network.py:1363]: Same comment here. I assume `bulk_create_path` is not used anywhere?

**Review by mikewiebe — COMMENTED** [review]
Overall, looks good.  Just a few comments.

File: plugins/action/dcnm_interface.py [hunk 2]
```diff
@@ -16,19 +16,10 @@
...
 __metaclass__ = type
 
-import copy
-import re
-import sys
-
 from ansible_collections.ansible.netcommon.plugins.action.network import (
     ActionModule as ActionNetworkModule,
 )
-from ansible.module_utils.connection import Connection
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/action/dcnm_interface.py:26]: We dont need Connection in action plugins

File: plugins/action/dcnm_inventory.py [hunk 3]
```diff
@@ -16,17 +16,9 @@
...
 __metaclass__ = type
 
-import copy
-import re
-import sys
-
 from ansible_collections.ansible.netcommon.plugins.action.network import (
     ActionModule as ActionNetworkModule,
 )
-from ansible.module_utils.connection import Connection
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/action/dcnm_inventory.py:26]: We dont need Connection in action plugins.

File: plugins/module_utils/network/dcnm/dcnm.py [hunk 4]
```diff
@@ -46,6 +46,7 @@ def validate_ip_address_format(type, item, invalid_params):
         except socket.error:
             invalid_params.append('{} : Invalid {} address syntax'.format(item, addr_type))
 
+
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/module_utils/network/dcnm/dcnm.py:49]: Yes. Flake8 expect 2 lines between functions and it expect 1 line between methods

File: plugins/modules/dcnm_interface.py [hunk 5]
```diff
@@ -1922,14 +1918,14 @@ def dcnm_intf_compare_elements (self, name, sno, fabric, ie1, ie2, k, state):
             e1 = ie1
             e2 = ie2
         else:
-            if (isinstance(ie1, unicode)):
-               e1 = ie1.encode('utf-8')
+            if (isinstance(ie1, unicode)):  # noqa
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/modules/dcnm_interface.py:1921]: I added this comment # noqa to make flake8 omit warnings in this line. Type unicode was supported in python 2.X versions. From python 3 onwards type unicode is not supported. flake8 with python3 does recognizes type unicode and throws warning.

File: plugins/modules/dcnm_network.py [hunk 6]
```diff
@@ -773,7 +772,6 @@ def get_want(self):
             networks = []
 
             net_deploy = net.get('deploy', True)
-            vlan_id = net.get('vlan_id', "")
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/modules/dcnm_network.py:776]: variable vlan_id is not used anywhere in this method/file apart from this line.

File: plugins/modules/dcnm_network.py [hunk 7]
```diff
@@ -1032,10 +1030,6 @@ def get_diff_merge(self, replace=False):
                     # Need to query DCNM to fetch next available networkId and use it here.
 
                     method = 'POST'
-                    result = dict(
```
[the remark this answers is not in the document]
Comment (praveenramoorthy) [reply plugins/modules/dcnm_network.py:1035]: This variable "result" is not used anywhere in this method/file after this definition.

File: plugins/modules/dcnm_network.py (hunk 1 above)
Comment (praveenramoorthy) [reply plugins/modules/dcnm_network.py:1363]: This variable "bulk_create_path" is not used anywhere in this method/file after this definition.

**Review by mikewiebe — APPROVED** [review]
LGTM

**Pull request merged**